README.md

```diff
@@ -44,6 +44,54 @@ module.exports = function (fastify, options, next) {
 npm run example -- -l info -w
 ```
 
+Send test messages to running server (running on localhost:3000 by default).
```

Comment:
I would not have such specific examples (e.g. with actual signatures / hmacs). These feel like they are more for your local testing? What we do want longer term in the README is how to _generate_ a signature or hmac for testing.

Reply:
I moved the specific examples to `TESTING_EXAMPLES.md` for now. We can remove this file later on.

Reply:
Removed testing file from repo